### PR TITLE
💄 Replace glamor

### PR DESCRIPTION
**Pull request by anncatton**
Replaces `glamor` with `emotion/react` and `emotion/styled`
Adds colour and dimension themes
Replaces inconsistent colour values, updates colours to overture style guide
Converts some classes to functional components so hooks can be used
*tried to reuse as much of the pre-existing object styles as possible, but emotion complained about some properties in camelcase so for these string styles were used (i.e. `wordBreak`, `flexDirection` etc)

### CODE REVIEW

File: src/theme/colors.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  accent_dark: '#003055', // DARK_BLUE
+  accent_1: '#e5edf3',
+};
+
+// light blues
+const secondary = {
+  secondary: '#4bc6f0', // LIGHT_BLUE
+  secondary_light: '#edf9fd',
+  secondary_dark: '#109ed9', // BLUE
+  secondary_accessible: '#0c7cac', // MEDIUM_BLUE
```
Comment (ciaranschutte) [inline src/theme/colors.ts:41]: i appreciate these comments !

**Comment by ciaranschutte** [comment]
@anncatton All looks good. I'm struggling a bit with how much "noise" emotion adds.
Looks like there's a mixed usage of Emotion + classic classnames which is fitting it in with some of the ui package component I think?

Read a cool article today. Not saying change it to this. But so far I like this idea.
https://medium.com/inturn-eng/naming-styled-components-d7097950a245
Namespacing styled components and have them in a seperate file. 
Benefits clearing up the actual React component file of extra code and better naming.
eg.
```
<S.Button> and <Button>
instead of
<StylingButton> <UnstyledButton>
```

**Review by ciaranschutte — APPROVED** [review]

File: src/theme/colors.ts (hunk 1 above)
Comment (anncatton) [reply src/theme/colors.ts:41]: oh haha those were for me as i was going through replacing the old colour scheme...i meant to delete them! but i'll keep the comments above each colour category for reference

**Comment by anncatton** [comment]
> @anncatton All looks good. I'm struggling a bit with how much "noise" emotion adds.
> Looks like there's a mixed usage of Emotion + classic classnames which is fitting it in with some of the ui package component I think?
> 
> Read a cool article today. Not saying change it to this. But so far I like this idea.
> https://medium.com/inturn-eng/naming-styled-components-d7097950a245
> Namespacing styled components and have them in a seperate file.
> Benefits clearing up the actual React component file of extra code and better naming.
> eg.
> 
> ```
> <S.Button> and <Button>
> instead of
> <StylingButton> <UnstyledButton>
> ```

I'll take a look!
Yeah, some of the classnames I wanted to double check if they were being used for any selection purposes before deleting ( i think i didn't finish this process, thanks for reminding me!) Some I added in this pr because overriding the semantic ui styles is a bit finicky, and previously there were a lot of `!important`s that did that, and I wanted to get rid of those. I'll do another run through to see which classNames are just cruft.

File: src/theme/colors.ts (hunk 1 above)
Comment (joneubank) [reply src/theme/colors.ts:41]: comments are useful, no reason to delte.

**Review by joneubank — APPROVED** [review]
Cool story bro.

**Review by joneubank — APPROVED** [review]

**Pull request merged**